subsys/logging/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ if(NOT CONFIG_LOG_MODE_MINIMAL)
     log_list.c
     log_core.c
     log_mgmt.c
+    log_cache.c
```

Comment:
Could you move this to the commit introducing `log_cache.c`? So `git bisect` would still work.